writethedocs/www
# Update Python from 3.8 to 3.9

### What

- Updates the project used Python version from 3.8 to 3.9
- Why? 3.8 has become unsupported, see: https://devguide.python.org/versions/#supported-versions

### How to test
- Follow the "Installing the project" [instructions](https://github.com/writethedocs/www?tab=readme-ov-file#prerequisites-for-generating-the-docs-locally).
- Make sure the projects runs on Python 3.9
- Check out the Python 3.9 [deprecated section](https://docs.python.org/3/whatsnew/3.9.html#deprecated) for breaking changes 
- All GitHub Actions (pipelines) should pass


----
📚 Documentation preview 📚: https://writethedocs-www--2271.org.readthedocs.build/
